onnxruntime/core/optimizer/constant_folding.cc

```diff
@@ -32,14 +34,12 @@ Status ConstantFolding::ApplyImpl(Graph& graph, bool& modified, int graph_level)
         // if the node output is in the graph output, we will get a graph with no nodes.
```

Comment:
Can we now relax the `node->ContainsSubgraph()`? Maybe not because we need the implicit inputs here and we still don't pass them to the execution frame?

Reply:
Picked up the comment from the next set of changes here to clarify.

The node with a subgraph cannot be executed here. The nodes in the subgraph are processed by the Recurse call above this condition, so they're not excluded.